### ISSUE TITLE
Double click (or more) needed to interact with some funcional elements in the interface

### ISSUE THREAD

**Issue by Bryceed** [ppy/osu#12511]
**Describe the bug:** Since "the native PC tablets settings" update, it's beign complicated to open menus, mark checkboxes etc. even with tablet option disabled. Some parts now working normally (as main menu burrons and beatmap list) but others still needing to click twice, thrice or even more to work.

**Screenshots or videos showing encountered issue:** 
https://user-images.githubusercontent.com/42657376/115503388-b8687600-a24c-11eb-9380-18a5d99d0fee.mp4

**osu!lazer version:** 
2021.416.0 (_latest_)

**Logs:**
[database.log](https://github.com/ppy/osu/files/6348367/database.log)
[network.log](https://github.com/ppy/osu/files/6348368/network.log)
[performance.log](https://github.com/ppy/osu/files/6348370/performance.log)
[runtime.log](https://github.com/ppy/osu/files/6348371/runtime.log)
[updater.log](https://github.com/ppy/osu/files/6348372/updater.log)

**Comment by peppy** [comment]
It looks like you are moving the cursor too much while clicking, and the game is determining that you are performing a drag. For more context, are you using a tablet or touch device here? Also, what resolution are you running the game at?

I believe right now the "click becomes drag" logic is done (incorrectly) using native pixels rather, which could mean that it is more likely to get in a bad state when running the game at a high resolution.

**Comment by peppy** [comment]
Actually on closer inspection, such operations should still be triggering a click, regardless of it turning into a drag (apparently an [issue in some cases?](https://github.com/ppy/osu-framework/issues/1382)).

Definitely looking to be a regression somewhere in input handling, so more details on your input devices would be appreciated.

**Comment by Bryceed** [comment]
Hi, 

I'm using a Wacom tablet with two monitors, both are 16:9 1366p (1366x768). 

Some months ago it's works normally but since that, it's happening this problem to click in menu. 
![image](https://user-images.githubusercontent.com/42657376/115504961-20b85700-a24f-11eb-8b9e-fea360dec202.png)
*_Btw is the sense that I ever used to play, half of the tablet surface._

**Comment by peppy** [comment]
Can you confirm that you have the in-game tablet handling set to off?

![20210421 151226 (dotnet)](https://user-images.githubusercontent.com/191335/115505159-fe0c5980-a2b3-11eb-8d24-daf906c56548.png)

**Comment by Bryceed** [comment]
As it can see in the video linked on the main report of this issue, when I try to turn on, still have that bug of "double cursors" but when turned off, happens this issue on clicking with some parts of the interface.

Ealier, when the update was installed, this issues happens even in main menu.

**Comment by Bryceed** [comment]
Testing with "Use Windows Ink" disabled off, the problem goes to the cursor sometimes not clicking or keep clicked, even with the pen away from the tablet.

https://user-images.githubusercontent.com/42657376/115506658-80176680-a251-11eb-9154-809a6f31150c.mp4

**Comment by peppy** [comment]
Right, I have a good idea of what's going on then. The actual plan was that you can leave the in-game tablet support enabled and *not* need to use the (somewhat haphazard Wacom implementation) but your window seems to be losing focus due to edge-of-screen clicks.

I know in stable we have windows ink disabling / taming code to handle edge cases like this, especially with incorrect clicks (already being tracked via https://github.com/ppy/osu-framework/issues/4258), which may help with this.

I think you can fix this issue for the time being by turning off "high precision mouse" and also turning off tablet support. Can you give that a try?

**Comment by Bryceed** [comment]
When I turn off both options (in-game and Windows Ink), the problem looks fixed for now!

Unfortunalely this lost the practicality to interact with both monitors just clicking the pen "in the other half" of the tablet, but at least it works. _An option to choose what monitor we want to play the game will be great too._

Thanks a lot for this fast reply!

**Comment by peppy** [comment]
For reference could you make a video showing the problem with multiple monitors? Is this when you're running the game windowed?

**Comment by Bryceed** [comment]
Basically the game considers all "adjacent free area in the tablet" that should be used for the other monitor as a "edge click" in the game (_but if needed I can record it anyway_).

**Comment by peppy** [comment]
It would still be very handy to have a video showing this, if/when you find time!
